packages/app/src/app/overmind/namespaces/live/actions.ts

```diff
@@ -36,7 +61,7 @@ export const roomJoined: AsyncAction<{
   effects.live.sendModuleStateSyncRequest();
   effects.vscode.openModule(state.editor.currentModule);
   effects.preview.executeCodeImmediately({ initialRender: true });
-  state.live.isLoading = false;
```

Comment:
Did we remove `state.live.isLoading`?

Reply:
Yeah, this is being set inside `live.initialize`, no reason to do it twice :-)